.ci/containers/terraform-vcr-tester/check_membership.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+PR_NUMBER=$1
+
+USER=$(curl -H "Authorization: token ${GITHUB_TOKEN}" \
+  "https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/${PR_NUMBER}" | jq -r .user.login)
+
+# Only run tests for safe users
+if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e megan07 -e paddycarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn -e rileykarson); then
```

Comment:
Can we have this list in a shared file someplace?

Reply:
Yeah, merging as-is to allow builds to pass that reference the new trigger. Will come back and refactor